### PR TITLE
doc: Update page layout

### PR DESCRIPTION
**Pull request by rodrigosiqueira**
Currently, we use the standard Sphinx template, which is excellent;
however, we finally have our official website, and it will be nice to
have something more user-friendly. For this reason, this commit
introduces a new Sphinx theme named book; this is part of issue #275.

Signed-off-by: Rodrigo Siqueira <rodrigosiqueiramelo@gmail.com>

### CODE REVIEW

**Review by kwy95 — CHANGES_REQUESTED** [review]
While testing this patch I noticed that the new theme forces sphinx version <4. This caused problems when generated a warning while generating the docs when I had a version of the docs already done with sphinx 4.0.2, which is what I was using before. Deleting the `build/` folder and running `./setup.sh --docs` again got rid of the issue.
With this in mind I think we should consider deleting a previous documentation build. But I do realize this is probably not a big issue.
I also added a couple of in line comments with more specific changes.

**Comment by rodrigosiqueira** [comment]
Nice catch @kwy95 !
I already update this PR.

File: setup.sh [hunk 1]
```diff
@@ -76,6 +79,13 @@ function check_dependencies()
     fi
     eval "sudo $cmd"
   fi
+
+  for package in "${pip_packages[@]}"; do
+    pip list | grep -F "$package" > /dev/null
+    [[ "$?" != 0 ]] && pip_package_list="$package $pip_package_list"
+  done
+  cmd="pip install $pip_package_list"
+  eval "$cmd"
```
Comment (kwy95) [inline setup.sh:88]: Shouldn't we ask the user before running this?

**Review by kwy95 — COMMENTED** [review]
Just the one question about user approval.

File: setup.sh (hunk 1 above)
Comment (rodrigosiqueira) [reply setup.sh:88]: Why?

Comment (kwy95) [reply setup.sh:88]: I mean, we ask the user before installing the other dependencies, why should this one be different?

Comment (rodrigosiqueira) [reply setup.sh:88]: Before installing a system dependency, we ask if the user wants to proceed; we then install everything, including this pip package. Also, pip installation in the userspace sounds like a routine task that nobody cares about. I can make it optional if you want, but I don't see the value in doing that.

Comment (kwy95) [reply setup.sh:88]: Ok, how about we move the for loop to line 72-73 and add `pip_package_list`  to the `ask_yN` prompt? Also you need to check if `pip_package_list` is not zero, as running `pip install` without arguments produces an error.
I left a comment suggestion with what I mean.

**Review by kwy95 — APPROVED** [review]

**Comment by rodrigosiqueira** [comment]
Merged, thanks

**Pull request closed without merging**